packages/syntax-floating-components/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  MutableRefObject,
+  ReactElement,
+  ReactNode,
+  useLayoutEffect,
+  useRef,
+  useMemo,
+} from "react";
+import classnames from "classnames";
+import {
+  Placement,
+  arrow,
+  flip,
+  useFloating,
+  shift,
+  offset,
+} from "@floating-ui/react";
+
+import styles from "./Tooltip.module.css";
+
+const defaultPlacement = "top";
+
+const Tooltip = ({
+  children,
+  offset: offsetValue = 7, // include arrow size which equals sqrt(5^2 + 5^2) = 7.07 ~ 7,
+  open,
+  placement: placementProp,
+  referenceEl,
+  zIndex: zIndexProp = 1100,
```

Comment:
@JFChoe let's remove the default zIndex, also, no need to rename it to `zIndexProp`

Reply:
Alright, I removed the zIndex prop in general